app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationsScreenContent.kt

```diff
@@ -108,6 +111,10 @@ fun ConversationsScreenContent(
         hiltViewModelScoped<ChangeConversationFavoriteVMImpl, ChangeConversationFavoriteVM, ChangeConversationFavoriteStateArgs>(
             ChangeConversationFavoriteStateArgs
         ),
+    removeConversationFromFolderViewModel: RemoveConversationFromFolderVM =
+        hiltViewModelScoped<RemoveConversationFromFolderVMImpl, RemoveConversationFromFolderVM, RemoveConversationFromFolderArgs>(
```

Comment:
question: does it need to be scoped? We provide all arguments when calling `removeFromFolder` function and don't provide any args that should distinguish instances, so there could even be a single instance of this view model for all `conversationsSource`s, so using regular `hiltViewModel` should be enough, at least in theory but maybe I'm missing something 🤔

Reply:
It doesn't need to be scoped but thanks to that we don't need to pass preview view model when we want to generate preview somewhere higher in composable tree